package.json

```diff
@@ -33,10 +33,10 @@
     "zone.js": "^0.6.23"
   },
   "devDependencies": {
+    "@angular/cli": "^1.0.0",
     "@angular/compiler-cli": "2.4.0",
```

Comment:
![screenshot from 2017-04-12 18-49-32](https://cloud.githubusercontent.com/assets/17461330/24978360/06bcbab8-1fee-11e7-9ca4-785db91c236a.png)


i googled the error and acc. to solution these files (  package.json and angular-cli.json ) need to be modified .  The thing u r using has been deprecated.